src/main/typescript/integrations/quench/registerBatches.ts

```diff
@@ -1,8 +1,8 @@
-import { CONSTANTS } from "../../constants.js";
-import type { HookParams } from "../../hooks/quench/ready.js";
-import registerPromptTests from "../../tests/applications/prompt.js";
+// import { CONSTANTS } from "../../constants.js";
```

Comment:
This file should be deleted instead of commented out.

Reply:
Oh, I knew I was forgetting something. Done.